### PR TITLE
Bug - Fix click propagation on ellipsis menus

### PR DESCRIPTION
**Pull request by BLoe**
## What does this PR do?

- Fixes a bug where clicks on the menu items propagated up to the parent element

For more information on our expectations for the PR process, see the
[code review principles doc](https://www.notion.so/pixiebrix/Code-Review-Principles-1ce7276b82a84d2a995d55ad85e1310d?pvs=4)

### CODE REVIEW

File: src/components/ellipsisMenu/EllipsisMenu.tsx [hunk 1]
```diff
@@ -134,28 +132,35 @@ const EllipsisMenu: React.FunctionComponent<EllipsisMenuProps> = ({
...
-        data-testid="ellipsis-menu-button"
-        onClick={(event) => {
-          event.stopPropagation();
-        }}
-      >
-        <FontAwesomeIcon icon={faEllipsisV} />
-      </MenuButton>
-    }
+  // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions,no-restricted-syntax -- Just stopping propagation, don't need accessibility
+  <div
```
Comment (grahamlangford) [inline src/components/ellipsisMenu/EllipsisMenu.tsx:136]: Did you double check that adding this div doesn't break styling anywhere? Might be better as a span (inline-block vs block)

Comment (BLoe) [reply src/components/ellipsisMenu/EllipsisMenu.tsx:136]: I'm not seeing any styling changes for document builder, mods page list item action button, or local integration list item action button, or the mod list items.

**Comment by mnholtz** [comment]
What issues was this causing? Not clear from PR description how this was caught.

**Comment by BLoe** [comment]
>What issues was this causing?

@mnholtz I can record a Loom in a little bit

File: src/components/ellipsisMenu/EllipsisMenu.tsx (hunk 1 above)
Comment (grahamlangford) [reply src/components/ellipsisMenu/EllipsisMenu.tsx:136]: Usually only comes up if used inside a flex box

**Review by grahamlangford — APPROVED** [review]

**Comment by mnholtz** [comment]
> any of the action menu items on a mod cause the mod's component "drawer" in the page editor sidebar to toggle open/closed as well when you click a menu item.

I don't think I can replicate what you're saying on cws version of the extension right now. Obviously not urgent, just curious what was happening

**Pull request merged**